From 0667a0934bafb626d3abd7c5eebdc93b287e47bc Mon Sep 17 00:00:00 2001
From: Stefan Schmidt <stefan.schmidt@huawei.com>
Date: Thu, 8 Jul 2021 15:24:02 +0200
Subject: [PATCH] Build system changes to be handled by yocto recipe

Leave iptables subdir out do be handled manually as it is not integrated
in autotools.

Upstream-Status: Inappropriate [other]

Signed-off-by: Stefan Schmidt <stefan.schmidt@huawei.com>
---
 src/usr/Makefile.am     | 3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/src/usr/Makefile.am b/src/usr/Makefile.am
index 7f4758ce..f6b1cdf0 100644
--- a/src/usr/Makefile.am
+++ b/src/usr/Makefile.am
@@ -1 +1,1 @@
-SUBDIRS = util nl argp siit nat64 iptables joold
+SUBDIRS = util nl argp siit nat64 joold
